packages/ui/src/domainModules/agent/components/AgentIdTextResult/index.js

```diff
@@ -106,14 +114,18 @@ class AgentIdTextResult extends Component {
     })
 
     if (textOnly) {
-      return <React.Fragment>{`${agentName} [${suffix}]`}</React.Fragment>
+      return (
+        <React.Fragment>
+          {`${agentName} ${disambiguatingDescriptionText} [${suffix}]`}
```

Comment:
this will lead to double space between agent name and suffix if there is no disambiguatingDescriptionText